config/software/cacerts.rb

```diff
@@ -18,29 +18,18 @@
 name "cacerts"
 
 # Date of the file is in a comment at the start, or in the changelog
-default_version "2014.09.03"
+default_version "2015.02.25"
 
-version "2014.09.03" do
-  source md5: "d7f7dd7e3ede3e323fc0e09381f16caf"
+version "2015.02.25" do
+  source :md5 => "5188ae76eaba5489f86a02156e1a647d"
+  source :url => "https://raw.githubusercontent.com/bagder/ca-bundle/d82fc46afaf6478aaa22989bcc2202ba7b72ad71/ca-bundle.crt"
 end
 
-version "2014.08.20" do
-  source :md5 => "c9f4f7f4d6a5ef6633e893577a09865e"
+version "2014.08.13" do
+  source :md5 => "f362813cd75967fa3096c00e5cf67914"
+  source :url => "https://raw.githubusercontent.com/bagder/ca-bundle/e9175fec5d0c4d42de24ed6d84a06d504d5e5a09/ca-bundle.crt"
 end
 
-version "2014.07.15" do
-  source :md5 => "fd48275847fa10a8007008379ee902f1"
-end
-
-version "2014.04.22" do
-  source :md5 => "9f92a0d9f605e227ae068e605f4c86fa"
-end
-
-version "2014.01.28" do
-  source :md5 => "5d108f8ab86afacc6663aafca8604dd3"
-end
```

Comment:
Yes, maybe; but, IMHO our breakage policy is a bit different for this repository.  Our goal is to provide a sensible set of software definitions that make it easy for Chef Software developers to get working, sensible software definitions that provide them with versions of the software that they can be confident shipping in a omnibus package without too much additional thought.  If they are using these older cert bundles, we should break them the next time they bump omnibus-software.  

Perhaps we should add a Recommend Use section to the README that would include something like this:

> Omnibus built projects can consume this library via bundler.  Your Gemfile.lock should be committed to your source code repository. Periodically, you should `bundle update omnibus-software` to pull in the latest software definition updates and resolve any problems that arise from version conflicts.

Perhaps in an delivery-centered world we can encourage people to float more aggressively, but right now I think the above is the most productive way to make use of this repo.  Lock to a version, update once early in a release cycle to pull in the latest version of stuff and give things time to stabilize.